VideoLocker/src/main/java/org/edx/mobile/view/custom/ECheckedTextView.java

```diff
@@ -0,0 +1,40 @@
+package org.edx.mobile.view.custom;
+
+import android.content.Context;
+import android.content.res.TypedArray;
+import android.graphics.Typeface;
+import android.util.AttributeSet;
+import android.widget.CheckedTextView;
+
+import org.edx.mobile.R;
+
+public class ECheckedTextView extends CheckedTextView {
+    public ECheckedTextView(Context context) {
+        super(context);
+    }
+
+    public ECheckedTextView(Context context, AttributeSet attrs) {
+        super(context, attrs);
+        processAttrs(context, attrs);
+    }
+
+    public ECheckedTextView(Context context, AttributeSet attrs, int defStyleAttr) {
+        super(context, attrs, defStyleAttr);
+        processAttrs(context, attrs);
+    }
+
+    private void processAttrs(Context context, AttributeSet attrs) {
+        if (isInEditMode()) return;
+        TypedArray a = context.getTheme().obtainStyledAttributes(
+                attrs, R.styleable.custom_view, 0, 0);
+        // Check for the font attribute and setup font
+        String fontFileName = a.getString(R.styleable.custom_view_font);
+        a.recycle();
+        if (fontFileName == null) {
+            // If font is not defined, then set default font
+            fontFileName = "OpenSans-Regular.ttf";
```

Comment:
>it gets the job done with minimal amount of lines written and code changes.

Thats what I loved about it and it does it without any extra library inclusion. So I'd still keep in mind for small apps on a tight deadline :wink: 

Neverthless, I've added a `relates-to` link with MA-1487 to an issue I reported:
https://openedx.atlassian.net/browse/MA-1403